src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partnerStatus = `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`;
```

Comment:
rename `partnerStatus -> partnerInfo` to improve var meaning because 'I am not married' is also partnerStatus